document gotcha around third party plugins (#11032)
hashicorp/packer: website/content/docs/commands/hcl2_upgrade.mdx

## Options

> - `-output-file` - Filename of the hcl2 generated config. Defaults to

Reviewer: `Filename of the Packer `hcl2_upgrade` generated HCL2 template...` seems less confusing to me.  These are either config files or template files, and since Packer does have a config file it seems reasonable to use template file to describe what is being converted.

Reviewer 2: ```suggestion
- `-output-file` - Filename of the hcl2 generated template. Defaults to
```

>   JSON_TEMPLATE.pkr.hcl; for example, if your template is called

Reviewer: template -> input-template

Reviewer 2: ```suggestion
  JSON_TEMPLATE.pkr.hcl; for example, if the file is called
```

>   "packerparty.json", the default output-file is "packerparty.json.pkr.hcl".

Reviewer: output-file -> output-template

Reviewer 2: ```suggestion
  "packerparty.json", the default output-template is "packerparty.json.pkr.hcl".
```

Reviewer 2: It's also a file ! I'd think this is okay
